bokehjs/src/coffee/models/renderers/glyph_renderer.coffee

```diff
@@ -52,6 +52,10 @@ export class GlyphRendererView extends RendererView
     @xscale = @plot_view.frame.xscales[@model.x_range_name]
     @yscale = @plot_view.frame.yscales[@model.y_range_name]
 
+    # compat, deprecated, to be removed
+    @xmapper = @xscale
+    @ymapper = @yscale
+
```

Comment:
you could have a getter and an underscore property "_has_warned" that gets updated?